src/Components/PostList.jsx

```diff
@@ -1,15 +1,39 @@
-import FakeArticleStub from "./ArticleStub.jsx";
+import DropDown from "./ArticleStub.jsx";
+import { useState, useEffect } from "react";
+import Accordion from "react-bootstrap/Accordion";
+import PropTypes from "prop-types";
 
 //TODo: update to take stubProp
```

Comment:
Safe to remove now that you added the prop?

Reply:
done